torch/_torch_docs.py

```diff
@@ -6656,6 +6656,30 @@ def merge_dicts(*dicts):
     tensor([ -24.8863, -771.4742])
 """.format(**common_args))
 
+add_docstr(torch.positive,
+           r"""
+positive(input, *, out=None) -> Tensor
```

Comment:
`positive(input) -> Tensor`

the current implementation does not support the out= argument